packages/cli/src/commands/setup/auth/providers/ethereum.js

```diff
@@ -31,7 +32,7 @@ export const config = {
 
     ethereum = new EthereumAuthClient({
       makeRequest,
-      debug: process.NODE_ENV === 'development',
+      debug: process.NODE_ENV !== 'development',
```

Comment:
Huh? 🙂

Reply:
You're right! Changed it back, Idk what I was thinking